watchtower/blob/justice_kit_packet.go

```diff
@@ -400,3 +415,222 @@ func (b *justiceKitPacketV0) decode(r io.Reader) error {
 
 	return nil
 }
+
+// justiceKitPacketV1 is lé Blob of Justice for taproot channels.
```

Comment:
lol TIL

Reply:
my two sats here: "le" is the French masculine word for "the". But because just "le" doesn't look very French, I assume the acute accent was added (and I believe all of this is from Sponge Bob if I'm not mistaken).